### PR TITLE
Add Service Worker Scope to Security Model Section

### PR DESCRIPTION
**Pull request by ayuishii**
- Add to authors list
- Add Service Worker Scope section to Security Model Section per TAG review feedback

### CODE REVIEW

File: explainer.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 *   Marijn Kruisselbrink - [mek@chromium.org](mailto:mek@chromium.org)
 *   Staphany Park - [staphany@chromium.org](mailto:staphany@chromium.org)
 *   Victor Costan - [pwnall@chromium.org](mailto:pwnall@chromium.org)
+*   Ayu Ishii - [ayui@chromium.org](mailto:ayui@chromium.org)
```
Comment (pwnall) [inline explainer.md]: nit: This list is alphabetically sorted. I think you get to go first! 😛

File: explainer.md [hunk 2]
```diff
@@ -506,6 +507,14 @@ The change events API exposed to documents does not accept URLs.
 Service workers may pass any URL under their scopes into the query or
 change event API.
 
+#### Service Worker Scope 
+
+The Cookie Store API does not change the access level for Service Workers.
+
+We think that Service Workers can currently effectively access the cookies of any URL under their scope. For example, a Service Worker could respond to any top-level request with an HTML document embedding an `<iframe>` pointing to the desired URL. When responding to the request for that URL, the Service Worker can respond with an HTML document containing a `<script>` that implements a postMessage API that proxies the Service Worker's access to the `document.cookie` API.
+
+For this reason we think that the API does not change the security properties of cookies on the Web Platform.
```
Comment (pwnall) [inline explainer.md]: I don't think we need this paragraph here. It's a good conclusion in a standalone message, but this section fits into a longer doc.

File: explainer.md [hunk 3]
```diff
@@ -506,6 +507,14 @@ The change events API exposed to documents does not accept URLs.
 Service workers may pass any URL under their scopes into the query or
 change event API.
 
+#### Service Worker Scope 
+
+The Cookie Store API does not change the access level for Service Workers.
+
+We think that Service Workers can currently effectively access the cookies of any URL under their scope. For example, a Service Worker could respond to any top-level request with an HTML document embedding an `<iframe>` pointing to the desired URL. When responding to the request for that URL, the Service Worker can respond with an HTML document containing a `<script>` that implements a postMessage API that proxies the Service Worker's access to the `document.cookie` API.
```
Comment (inexorabletash) [inline explainer.md]: Suggest removing "We think that" and "effectively".

And "implements a postMessage API" is a bit awkward. Maybe rewrite the last bit to: that proxies the Service Worker's access to the `document.cookie` API using `postMessage`.

Comment (ayuishii) [reply explainer.md]: :+1:

File: explainer.md (hunk 2 above)
Comment (ayuishii) [reply explainer.md]: :+1:

File: explainer.md (hunk 1 above)
Comment (ayuishii) [reply explainer.md]: :sweat_smile: :+1:

**Pull request merged**